src/server/getters/get-initial-reporters.ts

```diff
@@ -1,14 +1,23 @@
 import * as Knex from "knex";
+import Augur from "augur.js";
 import { Address } from "../../types";
-import { InitialReportersRow } from "../../types";
+import { queryModifier } from "./database";
+import { InitialReportersRow, UIInitialReporters } from "../../types";
 
-export function getInitialReporters(db: Knex, reporter: Address, redeemed: boolean|null|undefined, callback: (err: Error|null, result?: any) => void): void {
-  let query = db.select(["marketId", "reporter", "amountStaked", "initialReporter", "redeemed"]).from("initial_reports").where({ reporter });
+export function getInitialReporters(db: Knex, augur: Augur, reporter: Address, redeemed: boolean|null|undefined, withRepBalance: false, callback: (err: Error|null, result?: any) => void): void {
```

Comment:
Why is the `withRepBalance` parameter set to false?  Shouldn't it look more like the `redeemed` parameter instead (i.e., `withRepBalance: boolean|null|undefined`)?

I'm also not sure I understand why the balance would be different than the amount staked. Can you please explain this?

Reply:
`withRepBalance` normally isn't needed as `redeemed` is sufficient. Even though this will be used elsewhere my initial use case is for the escape hatch, which is weird because even if the initial reporter is not redeemed it can potentially have already been emptied through the emergency withdraw method, so I need to conditionally get the ones that actually have a remaining balance

Reply:
Ok, so when `augur.reporting.getInitialReporters` is called from the augur.js side, will it accept `withRepBalance` as a parameter, or will it always pass in `false` for `withRepBalance` when calling this getter?

Reply:
Oh actually I may have typoed that looking again. I'm pretty sure you're correct haha, will address